src/modules/pmc/src/lib/Pmc.cpp

```diff
@@ -18,9 +21,9 @@ Pmc::Pmc(unsigned int maxPayloadSizeBytes)
 int Pmc::RunCommand(const char* command, std::string* textResult, bool isLongRunning)
 {
     char* buffer = nullptr;
-    int status = ExecuteCommand(nullptr, command, true, true, 0, isLongRunning ? TIMEOUT_LONG_RUNNING : 0, &buffer, nullptr, PmcLog::Get()); 
+    int status = ExecuteCommand(nullptr, command, true, true, 0, isLongRunning ? TIMEOUT_LONG_RUNNING : 0, &buffer, nullptr, PmcLog::Get());
```

Comment:
>true, [](http://example.com/codeflow?start=56&length=5)

Why use true for the forJson argument? It replaces characters " and \ to spaces. Is this intended?

Reply:
good catch, changed to false.